lib/ohai/plugins/solaris2/network.rb

```diff
@@ -155,14 +155,12 @@ def full_interface_name(iface, part_name, index)
             break
           end
         end
-        if iface[ifn][:arp]
-          iface[ifn][:arp].each_key do |addr|
+        iface[ifn][:arp]&.each_key do |addr|
             if addr.eql?(iaddr)
               iface[ifn][:addresses][iface[ifn][:arp][iaddr]] = { "family" => "lladdr" }
               break
             end
           end
```

Comment:
I think this indentation is off now?

Reply:
Chefstyle is unhappy about this.

Reply:
Yeah that's me using Rubocop directly. I pushed it up again after the Chefstyle treatment.-